core/src/window_service.rs

```diff
@@ -164,7 +164,10 @@ impl ReceiveWindowStats {
 fn verify_shred_slot(shred: &Shred, root: u64) -> bool {
     match shred.shred_type() {
         // Only data shreds have parent information
-        ShredType::Data => blockstore::verify_shred_slots(shred.slot(), shred.parent(), root),
+        ShredType::Data => match shred.parent() {
+            Some(parent) => blockstore::verify_shred_slots(shred.slot(), parent, root),
```

Comment:
👍 ooft, good catch. Might want to backport this change for safety?

Reply:
will do